tests/apollo/test_printer.py

```diff
@@ -0,0 +1,22 @@
+import textwrap
+
+import strawberry
+from strawberry.apollo.schema_directives import CacheControl
+from strawberry.printer import print_schema
+
+
+def test_print_simple_cache_control():
+    @strawberry.type
+    class Query:
+        first_name: str = strawberry.field(directives=[CacheControl(max_age=10)])
```

Comment:
don't know if this test is necessary... Directives are tested somewhere else I think 🤔